hathor/p2p/states/ready.py

```diff
@@ -57,16 +64,29 @@ def __init__(self, protocol: 'HathorProtocol') -> None:
         # Minimum round-trip time among PING/PONG.
         self.ping_min_rtt: float = inf
 
+        # The last blocks from the best blockchain in the peer
+        self.best_blockchain: list[BlockInfo] = []
+
         self.cmd_map.update({
             # p2p control messages
             ProtocolMessages.PING: self.handle_ping,
             ProtocolMessages.PONG: self.handle_pong,
             ProtocolMessages.GET_PEERS: self.handle_get_peers,
             ProtocolMessages.PEERS: self.handle_peers,
-
             # Other messages are added by the sync manager.
         })
 
+        # if the peer has the GET-BEST-BLOCKCHAIN capability
+        if settings.CAPABILITY_GET_BEST_BLOCKCHAIN in protocol.capabilities:
+            # set the loop to get the best blockchain from the peer
+            self.lc_get_best_blockchain = LoopingCall(self.send_get_best_blockchain)
```

Comment:
Set `self.lc_get_best_blockchain: Optional[LoopingCall] = None` before the if block. So we can check whether it's none or not.

Reply:
Thanks!